### PR TITLE
Local quickstart cleanup

### PR DESCRIPTION
**Pull request by yvonneanne**
Following advice from #docs-working-group

### CODE REVIEW

File: modules/quickstart/pages/local-quickstart.adoc [hunk 1]
```diff
@@ -139,16 +136,16 @@ dfx start
 +
 Depending on your platform and local security settings, you might see a warning displayed.
 If you are prompted to allow or deny incoming network connections, click *Allow*.
-. Leave the terminal window that displays network operations open and switch your focus to the first terminal window where you created your new project.
+. Leave the terminal window that displays canister execution operations open and switch your focus to the first terminal window where you created your new project.
 +
-You perform the remaining steps in the terminal that doesn't display network operations.
+You perform the remaining steps in the terminal that doesn't display {IC} operations.
```
Comment (andreacerulli) [inline modules/quickstart/pages/local-quickstart.adoc:141]: Oh I think I missed these on mine 🙈

File: modules/quickstart/pages/local-quickstart.adoc [hunk 2]
```diff
@@ -273,16 +270,16 @@ dfx stop
...
 Here are some suggestions for where to go next:
 
-* link:../developers-guide/tutorials-intro{outfilesuffix}[Tutorials] to explore building simple applications using a local development environment.
+* link:../developers-guide/tutorials-intro{outfilesuffix}[Tutorials] to explore building simple dapps using a local canister execution environment.
 
-* link:network-quickstart{outfilesuffix}#convert-icp[Convert ICP tokens to cycles] if you have ICP tokens that you want to convert to cycles to enable you to deploy applications to the {IC} network.
+* link:network-quickstart{outfilesuffix}#convert-icp[Convert ICP tokens to cycles] if you have ICP tokens that you want to convert to cycles to enable you to deploy dapp to the {IC} blockchain.
 
-* link:network-quickstart{outfilesuffix}[Network deployment] if you have cycles and are ready to deploy an application to the {IC} network.
+* link:network-quickstart{outfilesuffix}[On-chain deployment] if you have cycles and are ready to deploy an application to the {IC} blockchain.
```
Comment (andreacerulli) [inline modules/quickstart/pages/local-quickstart.adoc:282]:
```suggestion
* link:network-quickstart{outfilesuffix}[On-chain deployment] if you have cycles and are ready to deploy an application to the {IC} blockchain mainnet.
```

**Review by andreacerulli — APPROVED** [review]
LGTM

File: modules/quickstart/pages/local-quickstart.adoc (hunk 1 above)
Comment (yvonneanne) [inline modules/quickstart/pages/local-quickstart.adoc:141]:
```suggestion
You perform the remaining steps in the terminal that doesn't display canister execution operations.
```

**Pull request merged**